.gitignore

```diff
@@ -19,4 +19,4 @@
 vendor/
 _output/
 /solo-kit-gen
-/doc-gen-test-*
+**/doc-gen-test-*
```

Comment:
what's the `**` syntax  mean?

Reply:
Any folder from root, because it now lives in a nested folder structure, it needs to be ignored there rather than just at the root